downstream/assemblies/platform/assembly-using-rhsso-operator-with-automation-hub.adoc

```diff
@@ -17,7 +17,7 @@ Use this Operator to create custom resources to automate Red Hat Single Sign-On
 
 This chapter describes the process to configure Red Hat Single Sign-On and integrate it with {PrivateHubName} when {PlatformNameShort} is installed on {OCPShort}.
 
-For more information on running operators on {OCPShort}, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/operators/index[Working with Operators in {OCPShort}] in the {OCPShort} product documentation.
+For more information on running operators on {OCPShort}, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{OCPLatest}/html/operators/index[Working with Operators in {OCPShort}] in the {OCPShort} product documentation.
```

Comment:
Links like this should go into the "Additional resources" section rather than inline. Putting links in additional resources has the following benefits:

- Avoids getting users off track / topic
- "Trains" users to always look for related info at the bottom of the topic. Creates a predictable pattern across RH docs, which improves overall navigation.
- Reduces maintenance cost for writing teams. Centralizing links is great because it makes it easier to update them and reduces the likelihood that writers overlook freshning up urls when updating topics.